frontend/interfaces/UserTypes.ts

```diff
@@ -1,14 +1,16 @@
+import { User } from "firebase/auth";
+
 export type UserId = string;
 
-interface AbstractUserData {
+export interface AbstractUserData {
```

Comment:
why are you exporting the `AbstractUserData`? isnt it abstract for a reason, where abouts are u exporting it to to use it?